src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/app.config

```diff
@@ -3,10 +3,6 @@
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
-        <assemblyIdentity name="System.Diagnostics.DiagnosticSource" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
```

Comment:
I pulled this one out manually -- it didn't auto-update, which was confusing, and it wasn't being used anyway.